src/WordPress/Hooks.php

```diff
@@ -82,47 +79,17 @@ public function initProxy()
 
     public function activate()
     {
-        $this->checkVersionCompatibility();
-        $this->checkDependenciesExist();
```

Comment:
I don't agree with `avoid functions in Hooks.php `. As long as the code is readable we can have many functions. Functions provide code documentation by default. 

In my opinion the code got relatively harder to read compared to the previous version still it's in a reasonable range to understand the code.